content/api/loaders.md

```diff
@@ -129,14 +167,14 @@ By default, loader results are cacheable. Call this method passing `false` to ma
 
 A cacheable loader must have a deterministic result, when inputs and dependencies haven't changed. This means the loader shouldn't have other dependencies than specified with `this.addDependency`. Most loaders are deterministic and cacheable.
```

Comment:
`this.cacheable` is set by default now. That might be worth mentioning. Check with Tobias.